Things of note:

While on the pi's network, you can check for the ip address of the pi and use that to connect.
Then you can send messages to and from.

For some weird reason, when we try to dynamically grab the ip address of the pi (just in case it has changed), the python script fails with a weird error.
	One solution is to give the pi a static IP, which shouldn't be an issue
